app/components/additional-fields-input.js

```diff
@@ -0,0 +1,48 @@
+import Ember from "ember";
+import config from "stock/config/environment";
+import { pluralize } from "ember-inflector";
+import _ from "lodash";
+
+export default Ember.TextField.extend({
+  tagName: "input",
```

Comment:
This file looks like much similar to other components.
Can we use them or refactor?
I think the only difference is classNameBinding which we can pass it to the component as parameter.

Reply:
@Darshan-Chauhan for sending data for update i am creating an object and other files has keyup, whichkey, keydown, function which we don't required over here